webui/react/Makefile

```diff
@@ -1,5 +1,6 @@
 swagger_client = "src/services/swagger"
-PROJECT_ROOT := "../../"
+export PROJECT_ROOT := "../../"
+export PUBLIC_URL ?= /det
```

Comment:
**nit:** `PUBLIC_PATH_PREFIX`? I usually think of url to include the protocol, host, port, path, etc.

Reply:
hmm i'm no sure if we can change that. I didn't pick that var name

Reply:
oh right right, nevermind! :D